pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -28,11 +28,17 @@ type SecurityPolicyEnforcer interface {
 	EnforceExpectedMountsPolicy(containerID string, spec *oci.Spec) error
 	EnforceMountPolicy(sandboxID, containerID string, spec *oci.Spec) error
 	ExtendDefaultMounts([]oci.Mount) error
+	EncodedSecurityPolicy() string
 }
 
 func NewSecurityPolicyEnforcer(state SecurityPolicyState) (SecurityPolicyEnforcer, error) {
 	if state.SecurityPolicy.AllowAll {
-		return &OpenDoorSecurityPolicyEnforcer{}, nil
+		if state.SecurityPolicy.Containers.Length > 0 || len(state.SecurityPolicy.Containers.Elements) > 0 {
```

Comment:
Is there a way to guarantee that `Containers.Length` will always equal `len(Containers.Elements)`?

Reply:
I'm planning on working on the JSON unmarshaller that'll have these validations. TOML unmarshaller is terrible, so I don't want to deal with that lol.